src/templates/article.template.tsx

```diff
@@ -36,11 +33,11 @@ const siteQuery = graphql`
   }
 `;
 
-const Article: Template = ({ pageContext, location }) => {
-  const contentSectionRef = useRef<HTMLElement>(null);
+const Article = ({ pageContext, location, children }) => {
+  const contentSectionRef = useRef(null);
 
-  const [hasCalculated, setHasCalculated] = useState<boolean>(false);
```

Comment:
for some reason this throws `boolean` not found. will consider fixing it in a future release